service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerSubmittedTest.java

```diff
@@ -99,63 +106,50 @@ void shouldBuildNotificationTemplatesWithCompleteValues() throws Exception {
         verify(notificationClient, never()).sendEmail(
             HMCTS_COURT_SUBMISSION_TEMPLATE,
             "FamilyPublicLaw+ctsc@gmail.com",
-            completeHmctsParameters,
+            expectedHmctsParameters,
             CASE_REFERENCE.toString());
     }
 
     @Test
     void shouldBuildNotificationTemplatesWithValuesMissingInCallback() throws Exception {
         CaseDetails caseDetails = enableSendToCtscOnCaseDetails(NO);
-
-        Map<String, Object> expectedHmctsParameters = getInCompleteParameters()
-            .put("court", FAMILY_COURT)
-            .build();
-
-        Map<String, Object> expectedCafcassParameters = getInCompleteParameters()
-            .put("cafcass", CAFCASS_COURT)
-            .build();
-
         postSubmittedEvent(caseDetails);
 
         verify(notificationClient).sendEmail(
             HMCTS_COURT_SUBMISSION_TEMPLATE,
             HMCTS_ADMIN_EMAIL,
-            expectedHmctsParameters,
+            getExpectedIncompleteHmctsParameters(),
```

Comment:
If the aim was to reduce the number of times we are creating the object, by calling the method twice it is still being created twice